src/coreclr/nativeaot/System.Private.CoreLib/src/System/ModuleHandle.cs

```diff
@@ -62,7 +62,7 @@ public int MDStreamVersion
         {
             get
             {
-                throw new PlatformNotSupportedException();
```

Comment:
This property is not supported by native AOT runtime. `PlatformNotSupportedException` is correct behavior. It is not correct to change to return -1.